test/box-luatest/gh_6403_space_pairs_luafun_integration_test.lua

```diff
@@ -0,0 +1,44 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+-- Both engines to check if both luac and ffi implementations work correctly
```

Comment:
Nit: Comment length is more than 66 symbols.
Nit: Missed dot at the end of the sentence.

Typo: s/ffi/FFI/
Typo: s/luac/LuaC/